.circleci/config.yml

```diff
@@ -82,3 +92,9 @@ workflows:
           filters:
             branches:
               only: staging
+      - deploy_production:
```

Comment:
Should we have manual approval for production?

Reply:
Good idea.